[2021-04-18 17:00:14,120] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: fetch_kc_crime_data-v0.1.download_latest_crime_data 2021-04-18T17:00:02.895656+00:00 [queued]>
[2021-04-18 17:00:14,176] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: fetch_kc_crime_data-v0.1.download_latest_crime_data 2021-04-18T17:00:02.895656+00:00 [queued]>
[2021-04-18 17:00:14,178] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-18 17:00:14,178] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-04-18 17:00:14,179] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-18 17:00:14,192] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): download_latest_crime_data> on 2021-04-18T17:00:02.895656+00:00
[2021-04-18 17:00:14,202] {standard_task_runner.py:52} INFO - Started process 971 to run task
[2021-04-18 17:00:14,207] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'fetch_kc_crime_data-v0.1', 'download_latest_crime_data', '2021-04-18T17:00:02.895656+00:00', '--job-id', '13', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/fetch_kc_crime_data-v0.1.py', '--cfg-path', '/tmp/tmpbodyh24t', '--error-file', '/tmp/tmp6bvjwfuh']
[2021-04-18 17:00:14,212] {standard_task_runner.py:77} INFO - Job 13: Subtask download_latest_crime_data
[2021-04-18 17:00:14,318] {logging_mixin.py:104} INFO - Running <TaskInstance: fetch_kc_crime_data-v0.1.download_latest_crime_data 2021-04-18T17:00:02.895656+00:00 [running]> on host 9651c4896f9f
[2021-04-18 17:00:14,415] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=fetch_kc_crime_data-v0.1
AIRFLOW_CTX_TASK_ID=download_latest_crime_data
AIRFLOW_CTX_EXECUTION_DATE=2021-04-18T17:00:02.895656+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-04-18T17:00:02.895656+00:00
[2021-04-18 17:00:14,485] {logging_mixin.py:104} INFO - Min Window: 2021-01-01T00:00:00.000
[2021-04-18 17:00:14,486] {logging_mixin.py:104} INFO - Max Window: 2021-04-18T17%:00:10.955
[2021-04-18 17:00:14,488] {logging_mixin.py:104} INFO - {'$where': "report_date BETWEEN '2021-01-01T00:00:00.000' and '2021-04-18T17%:00:10.955'"}
[2021-04-18 17:00:15,032] {logging_mixin.py:104} INFO - Error in API call
[2021-04-18 17:00:15,038] {python.py:118} INFO - Done. Returned value was: None
[2021-04-18 17:00:15,105] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=fetch_kc_crime_data-v0.1, task_id=download_latest_crime_data, execution_date=20210418T170002, start_date=20210418T170014, end_date=20210418T170015
[2021-04-18 17:00:15,261] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-04-18 17:00:15,306] {local_task_job.py:146} INFO - Task exited with return code 0
